packages/fluentui/docs/src/examples/components/Loader/Variations/LoaderExampleSecondary.tsx

```diff
@@ -0,0 +1,16 @@
+import { Loader, Flex } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const LoaderExampleSecondary: React.FC = () => (
+  <Flex
+    style={{
+      backgroundColor: 'rgb(98, 100, 167)',
```

Comment:
```suggestion
      backgroundColor: siteVariables.token,
```

Can we use a token from a theme instead? We can grab it via `ProviderConsumer`.

This looks strange in high contrast:

![image](https://user-images.githubusercontent.com/14183168/126780271-5e03e80c-ac8a-4e0f-93e1-526df21805e4.png)